src/meta/src/rpc/metrics.rs

```diff
@@ -77,13 +83,13 @@ pub struct MetaMetrics {
     /// The smallest epoch that is being pinned.
     pub min_pinned_epoch: IntGauge,
     /// The number of SSTs in each level
-    pub level_sst_num: IntGaugeVec,
+    pub level_sst_num: LabelGuardedIntGaugeVec<1>,
     /// The number of SSTs to be merged to next level in each level
-    pub level_compact_cnt: IntGaugeVec,
+    pub level_compact_cnt: LabelGuardedIntGaugeVec<1>,
     /// The number of compact tasks
-    pub compact_frequency: IntCounterVec,
+    pub compact_frequency: LabelGuardedIntCounterVec<4>,
     /// Size of each level
-    pub level_file_size: IntGaugeVec,
+    pub level_file_size: LabelGuardedIntGaugeVec<1>,
     /// Hummock version size
     pub version_size: IntGauge,
```

Comment:
How to refactor IntGauge with LabelGuardedIntGauge<N>? There are no LabelGuardedIntGauge::\<N\>::new or a macro like `register_guarded_int_gauge` to build it. cc @tabVersion @wenym1.

Reply:
The `LabelGuardedXxx` is only used for `XxxVec`. The `IntGauge` without any label does not need to use as a label guarded one.